package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "chai-subset",
-  "version": "1.0.1",
-  "description": "Object properties matcher for Chai",
+  "name": "signavio-chai-subset",
```

Comment:
1. In case if you need to use fork for some time until i'll merge you can point to github in your package.json, like
```
{
 "decDependencies": {
  "chai-subset": "signavio/chai-subet"
 }
}
```
so it will install package from GitHub, so there is no need to publish forks to npm